src/App.cpp

```diff
@@ -60,8 +55,8 @@ int main(int argc, const char * argv[]) {
 
   run();
   
-  // Print how much objects were created during app running, and what have left-probably leaked
-  // Disable object counting for release builds using '-D OATPP_DISABLE_ENV_OBJECT_COUNTERS' flag for better performance
+  /* Print how much objects were created during app running, and what have left-probably leaked */
```

Comment:
Let's keep the single lines comments or properly use a multiline 

```suggestion
  // Print how much objects were created during app running, and what have left-probably leaked
```